server/planning/planning/planning.py

```diff
@@ -1005,9 +1005,10 @@ def delete_assignments_for_coverages(self, coverages, notify=True):
                     }
                 )
             except SuperdeskApiError:
-                logger.error("Failed to delete assignment {}".format(assign_id))
+                logger.error("There is a assignment '{}' is in progress".format(assign_id))
```

Comment:
Sorry, I meant that the `SuperdeskApiError` could be raised by many different places in our code (`planning`, `assignment`, `archive` or `delivery` services/resources).

It might be good if we pass the actual error to the end user, as to provide a more accurate reason why the action failed (still displaying them as warnings though, as the original action succeeded)